perseo-main/src/test/java/com/telefonica/iot/perseo/LogLevelServletTest.java

```diff
@@ -29,13 +29,16 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  *
  * @author brox
  */
 public class LogLevelServletTest {
-
+	
+	private static final Logger logger = LoggerFactory.getLogger(LogLevelServletTest.class);
```

Comment:
Same here (and last time I mention :D, please do a general review in all files)-

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823